src/hooks/use-canvas.ts

```
@@ -759,4 +759,244 @@ export const useCanvas = () => {
+            ctx.stroke()
+            break
+
+        case 'arrow':
+            ctx.strokeStyle = shape.stroke || '#ffffff'
+            ctx.lineWidth = shape.strokeWidth || 2
+            ctx.beginPath()
+            ctx.moveTo(shape.startX - frameX, shape.startY - frameY)
+            ctx.lineTo(shape.endX - frameX, shape.endY - frameY)
+            ctx.stroke()
+            
+            const headLength = 10
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)